### PR TITLE
SSH Key Instructions

### LINKED ISSUE #7886: Generate and Add SSH Keys Doc Update
Re: [Generate and Add SSH Keys](https://pantheon.io/docs/ssh-keys)

Priority: Medium

## Issue Description:

Windows and unix-based systems have overlapping instructions and unclear prompts. PR #7302 introduced Windows versions of some steps, sometimes these are correctly introduced as separate steps and sometimes not. Sometimes they are in windows bash samples and sometimes not.

## Suggested Resolution
Review document to ensure these are clearly separated. 

I noticed the steps here are presented as if you first create the file on Windows and then modify on a Unix based system: 
https://pantheon.io/docs/ssh-keys#connections-fail-with-no-matching-host-key-type-found-their-offer-ssh-rsa

And here on step 4 the prompt is broken into a bunch of pieces and doesn't have the Windows command prompt style:
https://pantheon.io/docs/ssh-keys#generate-ssh-key

There may be other issues

### PR DESCRIPTION
**Pull request by whitneymeredith**
Closes #7886 

## Summary

**[Generate SSH Key](https://docs.pantheon.io/ssh-keys#generate-ssh-key)** - Added separate instructions for MacOS, Linux, and Windows users.

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/old-path/` => `/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by whitneymeredith** [comment]
@jms-pantheon Can you please take a look at this when you get a moment?

**Comment by jms-pantheon** [comment]
Hi @whitneymeredith thanks for re-organizing this one, it is much clearer. One question, since the steps for Mac and Linux are the same should we combine those two?

**Review by wordsmither — APPROVED** [review]
lgtm

**Comment by jms-pantheon** [comment]
@wordsmither Looks like my one comment about the output lines from my review was missed

**Comment by wordsmither** [comment]
@jms-pantheon , what comment are you referring to?  I'm not seeing one...

File: source/content/ssh-keys.md [hunk 1]
```diff
@@ -12,91 +12,133 @@ product: [--]
...
+
+1. Copy the contents of `~/.ssh/id_rsa.pub` to your clipboard after the files are created.
+
+   ```bash{promptUser: winshell}
+   type .ssh\id_rsa.pub
+   ```
+
+1.  Run `start-ssh-agent` to start the SSH agent. The output confirms the agent has started. Enter the passphrase, if it was previously set.
+
+      ```bash{promptUser: winshell}
```
[the remark this answers is not in the document]
Comment (jms-pantheon) [reply source/content/ssh-keys.md:99]: resolving this as it will be fixed elsewhere

**Pull request merged**